src/main/scala/io/iohk/ethereum/jsonrpc/server/http/BasicJsonRpcHttpServer.scala

```diff
@@ -11,7 +11,7 @@ import io.iohk.ethereum.utils.Logger
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.util.{Failure, Success}
 
-class BasicJsonRpcHttpServer(
+abstract class BasicJsonRpcHttpServer(
```

Comment:
I think that maybe we can invert the next class name: "BasicJsonRpcHttpServer" with "JsonRpcHttpServer". What do you think?

Reply:
I agree!